Makefile

```diff
@@ -33,7 +33,6 @@ APP = gatekeeper
 SRCS-y := main/main.c
 
 # Functional blocks.
-SRCS-y += catcher/main.c
```

Comment:
Something is wrong here.

This Makefile has no entry for BP, which is removed in the following patch. Moreover, the following patch seems to add back the catch entry. The previous patch doesn't show an entry for BP as well.